docs/3-building-blocks-of-oop-part-2/hometask/src/Item.ts

```diff
@@ -1,11 +1,75 @@
-import { Comparable } from './Comparable';
+import { Comparable } from "./Comparable";
 
 let id = 0;
 
 export abstract class Item implements Comparable<Item> {
-    public compareTo(other: Item): number {
-        // your code goes here
+  id: number;
+  value: number;
+  weight: number;
+  name: string;
+
+  constructor(name: string, value: number, weight: number) {
+    this.name = name;
+    this.value = value;
+    this.weight = weight;
+    this.id = id;
+    id++;
```

Comment:
let's replace lines 15-16 with: `this.id = id++;`
In such case the value of `id` will be equal to the number of items.
Current implementation results in wrong value of `numberOfItems`

Reply:
done